clients/src/main/java/org/apache/kafka/clients/admin/ListShareGroupOffsetsSpec.java

```diff
@@ -47,9 +46,10 @@ public ListShareGroupOffsetsSpec topicPartitions(Collection<TopicPartition> topi
 
     /**
      * Returns the topic partitions whose offsets are to be listed for a share group.
+     * {@code null} indicates that offsets of all partitions of the group are to be listed.
      */
     public Collection<TopicPartition> topicPartitions() {
```

Comment:
What do we do elsewhere in the admin for "all" vs "none"?

Reply:
Not sure, but this is precisely the same behaviour as `ListConsumerGroupOffsetsSpec`. Essentially, if you do `new ListShareGroupOffsetsSpec()` you get all of them. If you do `new ListShareGroupOffsetsSpec().topicPartitions(....)` then you get a smaller list. There's no way to ask for none.